resources/views/date/gig/listAttendances.blade.php

```diff
@@ -49,21 +49,32 @@
+                                                                $userid = $user->id;
+                                                                $mark = '&nbsp;<i class="far fa-comment comment-toggle" id="comment-toggle-'.$userid.'-'.$gigid.'" title="'.$comment.'"></i>';
+                                                                $mark .= '<div id="comment-'.$userid.'-'.$gigid.'"" style="display: none">'.$comment.'</div>';
+                                                                $mark .= '<script type="text/javascript">$(document).ready(function(){$("#comment-toggle-'.$userid.'-'.$gigid.'").click(function(){$("#comment-'.$userid.'-'.$gigid.'").toggle();});});</script>';
+                                                            }
+                                                            else{
+                                                                $mark = '';
+                                                            }
+                                                        ?>
                                                         @if($gig->isAttending($user) == 'yes')
-                                                            <td class="attending" title="{{ $comment }}">
+                                                            <td class="attending">
                                                                 <i class="fa fa-check"></i>{!! $mark !!}
                                                             </td>
                                                         @elseif($gig->isAttending($user) == 'maybe')
-                                                            <td class="maybe-attending" title="{{ $comment }}">
+                                                            <td class="maybe-attending">
                                                                 <i class="fa fa-question"></i>{!! $mark !!}
                                                             </td>
                                                         @elseif($gig->isAttending($user) == 'no')
-                                                            <td class="not-attending" title="{{ $comment }}">
+                                                            <td class="not-attending">
                                                                 <i class="fa fa-times"></i>{!! $mark !!}
                                                             </td>
                                                         @else
-                                                            <td class="unanswered" title="{{ $comment }}">
+                                                            <td class="unanswered">
                                                                 <i class="fa fa-minus"></i>{!! $mark !!}
                                                             </td>
                                                         @endif
```

Comment:
I never liked the code-flow with $mark and it looks even worse with these changes. This is a great chance to get rid of the variable $mark and instead read "attending"/"maybe-attending"/"not-attending"/"unanswered" from a variable.
If you don't want to rework this (which is understandable), I'll do it.

Reply:
changed it. I'm waiting for your approval